integrations/sentry/clients/sentry.py

```diff
@@ -17,30 +20,61 @@ def __init__(
         self.client = http_async_client
         self.client.headers.update(self.base_headers)
 
+    def get_next_link(self, link_header: str) -> str:
```

Comment:
can you add link and a comment regarding how the data is returned resulting you to do that parsing